### PR TITLE
Add the slide for integration with ArgoCD

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>

### CODE REVIEW

**Comment by chenz4027** [comment]
FYI @mprahl

File: content/en/blog/_index.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ weight: 6
 
 <!-- spellchecker-enable -->
 
+## Oct 26, 2022
+
+[KubeCon NA 2022 - OCM Multicluster App & Config Management](/kubecon-na-2022-ocm-multicluster-app-and-config-management.pdf) authored by [Maggie Chen](https://github.com/chenz4027) and [Matt Prahl](https://github.com/mprahl)
+
```
Comment (dhaiducek) [inline content/en/blog/_index.md:15]: Would you please update [content/zh/blog/_index.md](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/blob/main/content/zh/blog/_index.md) also?
Looks like the Placement blog was missed there if you wouldn't mind adding that also.

Comment (chenz4027) [reply content/en/blog/_index.md:15]: oh sure, will do. Thanks @dhaiducek

Comment (chenz4027) [reply content/en/blog/_index.md:15]: updated

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**